### DISCUSSION TITLE
Chpp App Promotion

### DISCUSSION THREAD

**Discussion by wsbrenk** [ho-dev/HattrickOrganizer#1983] [General]
https://www82.hattrick.org/Forum/Read.aspx?t=17569901&n=1&v=0

i added our project to the mentioned ht thread: https://www82.hattrick.org/Forum/Read.aspx?t=17569863&n=1&v=0&mr=0

I'm curious to see what comes out of it for our project. 

@tychobrailleur Any good ideas how we could use it? I would like to suggest you as our speaker as my English is rather miserable.

**Comment by tychobrailleur** [comment]
> @tychobrailleur Any good ideas how we could use it? I would like to suggest you as our speaker as my English is rather miserable.

I guess this is mostly promoting HO to the wider community, right?  I do find your English quite excellent, but not to worry, happy to be the spokesperson of the project. ;-)
